tao_compiler/mlir/disc/disc_compiler.cc

```diff
@@ -425,6 +425,14 @@ LogicalResult LowerHLOToLLVM(ModuleOp m, const DISCLoweringOptions& options) {
   pm.addNestedPass<FuncOp>(
       disc_ral::createDiscUnhandledAtomicRMWConverterPass());
   pm.addNestedPass<FuncOp>(disc_ral::createDiscInputInlineFusionPass());
+  bool mem_intensive_opt_experimental = false;
+  tensorflow::ReadBoolFromEnvVar("DISC_MEM_INTENSIVE_OPT_EXPERIMENTAL", false,
```

Comment:
I saw some other PR use this env, maybe we need a global function to check `DISC_MEM_INTENSIVE_OPT_EXPERIMENTAL ` ?

Reply:
It's better to let each feature having it's own flag ... It is time to take a look these flags and do some 'feature engineering' ? :P

Reply:
`tensorflow::ReadBoolFromEnvVar` is already a wrapper. Another wrapper based on that does not save much code.